web/hooks/use-likes.ts

```diff
@@ -21,6 +23,23 @@ export const useLikesOnContent = (
         .eq('content_type', contentType)
         .eq('content_id', contentId)
     ).then(({ data }) => setLikes(data))
+    
+    const handleCommentUpdate = (msg: ServerMessage<'comment_update'>) => {
+      if (msg.data && msg.data.comment_id === contentId) {
+        setLikes(msg.data.likes)
+      }
+    }
+    
+    const subscription = useApiSubscription({
+      topics: [`comment:${contentId}`],
+      onBroadcast: handleCommentUpdate,
+    })
+    
+    return () => {
+      if (subscription) {
+        subscription.unsubscribe()
+      }
+    }
   }, [contentType, contentId])
```

Comment:
Ah, I just realized I've led you wrong. We need to get all the reactions here (which include who performed the like). That means we need to send a websocket event for `new-like` when each like is created in reaction.ts.